### PR TITLE
EWPP-2358: Drupal 9.4 support.

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "_readme": [
         "Explicit requirement of behat/behat module due to PHP8 compatibility.",
         "Explicit requirement of behat/mink-extension to assure lower compatibility.",
-        "Explicit requirement for egulias/email-validator due to https://www.drupal.org/project/drupal/issues/3061074#comment-14300579. It can be removed when Drupal core 9.2 support is droppped."
+        "Explicit minimum version requirement for fabpot/goutte due to lower version doesn't work with Drupal 9.3."
```
Comment (22Alexandra) [inline composer.json:69]:
```suggestion
        "Explicit minimum version requirement for fabpot/goutte due to Drupal 9.3 compatibility."
```

File: modules/oe_media_avportal/config/install/views.view.av_portal_entity_browsers.yml [hunk 2]
```diff
@@ -11,7 +11,6 @@ description: ''
 tag: ''
 base_table: avportal
 base_field: ''
-core_version_requirement: ^9.2
```
Comment (22Alexandra) [inline modules/oe_media_avportal/config/install/views.view.av_portal_entity_browsers.yml:14]: Why removing it entirely?

File: modules/oe_media_avportal/tests/modules/oe_media_avportal_test/config/install/views.view.av_portal_entity_browser_test.yml [hunk 3]
```diff
@@ -11,7 +11,6 @@ description: ''
 tag: ''
 base_table: avportal
 base_field: ''
-core_version_requirement: ^9.2
```
Comment (22Alexandra) [inline modules/oe_media_avportal/tests/modules/oe_media_avportal_test/config/install/views.view.av_portal_entity_browser_test.yml:14]: Why removing it entirely?

File: modules/oe_media_avportal/tests/modules/oe_media_avportal_test/config/install/views.view.av_portal_test.yml [hunk 4]
```diff
@@ -10,7 +10,6 @@ description: ''
 tag: ''
 base_table: avportal
 base_field: ''
-core_version_requirement: ^9.2
```
Comment (22Alexandra) [inline modules/oe_media_avportal/tests/modules/oe_media_avportal_test/config/install/views.view.av_portal_test.yml:13]: Why removing it entirely?

**Review by 22Alexandra — CHANGES_REQUESTED** [review]

File: modules/oe_media_avportal/config/install/views.view.av_portal_entity_browsers.yml (hunk 2 above)
Comment (yenyasinn) [reply modules/oe_media_avportal/config/install/views.view.av_portal_entity_browsers.yml:14]: Do we use core_version_requirement in configuration files?

File: modules/oe_media_avportal/tests/modules/oe_media_avportal_test/config/install/views.view.av_portal_entity_browser_test.yml (hunk 3 above)
Comment (yenyasinn) [reply modules/oe_media_avportal/tests/modules/oe_media_avportal_test/config/install/views.view.av_portal_entity_browser_test.yml:14]: Do we use core_version_requirement in configuration files?

File: modules/oe_media_avportal/tests/modules/oe_media_avportal_test/config/install/views.view.av_portal_test.yml (hunk 4 above)
Comment (yenyasinn) [reply modules/oe_media_avportal/tests/modules/oe_media_avportal_test/config/install/views.view.av_portal_test.yml:13]: Do we use core_version_requirement in configuration files?

File: modules/oe_media_avportal/config/install/views.view.av_portal_entity_browsers.yml (hunk 2 above)
Comment (22Alexandra) [reply modules/oe_media_avportal/config/install/views.view.av_portal_entity_browsers.yml:14]: Right.

**Review by 22Alexandra — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**